### PR TITLE
Update Wrapcheck to v2.3

### PR DESCRIPTION
**Pull request by hn8**
Update `ignoreSigs` default, add `ignorePackageGlobs`

https://github.com/tomarrell/wrapcheck/compare/v2.2.0...v2.3.0

### CODE REVIEW

File: .golangci.example.yml [hunk 1]
```diff
@@ -616,6 +616,9 @@ linters-settings:
       - .Wrap(
       - .Wrapf(
       - .WithMessage(
+      - .WithMessagef(
+      - .WithStack(
+    ignorePackageGlobs: []
```
Comment (ldez) [inline .golangci.example.yml:621]:
```suggestion
    ignorePackageGlobs:
        - encoding/*
        - github.com/pkg/*
```

**Review by ldez — CHANGES_REQUESTED** [review]
Hello,

the file `.golangci.example.yml` doesn't describe the default values but it provides an example of the linter's configuration.

**Review by ldez — APPROVED** [review]
LGTM

**Comment by tomarrell** [comment]
Thanks a bunch for sorting this.

**Review by tomarrell — APPROVED** [review]

**Pull request merged**